src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/servlet/ServletConstants.java

```diff
@@ -16,30 +16,20 @@
 public class ServletConstants {
 
     /* @WEBListener */
-    public static final String WEB_LISTENER = "WebListener";
     public static final String WEB_LISTENER_FQ_NAME = "jakarta.servlet.annotation.WebListener";
-    public static final String SERVLET_CONTEXT_LISTENER = "ServletContextListener";
     public static final String SERVLET_CONTEXT_LISTENER_FQ_NAME = "jakarta.servlet.ServletContextListener";
-    public static final String SERVLET_CONTEXT_ATTRIBUTE_LISTENER = "ServletContextAttributeListener";
     public static final String SERVLET_CONTEXT_ATTRIBUTE_LISTENER_FQ_NAME = "jakarta.servlet.ServletContextAttributeListener";
-    public static final String SERVLET_REQUEST_LISTENER = "ServletRequestListener";
     public static final String SERVLET_REQUEST_LISTENER_FQ_NAME = "jakarta.servlet.ServletRequestListener";
-    public static final String SERVLET_REQUEST_ATTRIBUTE_LISTENER = "ServletRequestAttributeListener";
     public static final String SERVLET_REQUEST_ATTRIBUTE_LISTENER_FQ_NAME = "jakarta.servlet.ServletRequestAttributeListener";
-    public static final String HTTP_SESSION_LISTENER = "HttpSessionListener";
     public static final String HTTP_SESSION_LISTENER_FQ_NAME = "jakarta.servlet.http.HttpSessionListener";
-    public static final String HTTP_SESSION_ATTRIBUTE_LISTENER = "HttpSessionAttributeListener";
     public static final String HTTP_SESSION_ATTRIBUTE_LISTENER_FQ_NAME = "jakarta.servlet.http.HttpSessionAttributeListener";
-    public static final String HTTP_SESSION_ID_LISTENER = "HttpSessionIdListener";
     public static final String HTTP_SESSION_ID_LISTENER_FQ_NAME = "jakarta.servlet.http.HttpSessionIdListener";
 
     /* @WEBServlet */
-    public static final String WEB_SERVLET = "WebServlet";
     public static final String WEB_SERVLET_FQ_NAME = "jakarta.servlet.annotation.WebServlet";
     public static final String HTTP_SERVLET = "HttpServlet";
 
     /* @WEBFilter */
-    public static final String WEBFILTER = "WebFilter";
     public static final String WEBFILTER_FQ_NAME = "jakarta.servlet.annotation.WebFilter";
     public static final String FILTER = "Filter";
```

Comment:
I was curious why this constant wasn't removed but can see that it's being used in an error message. That's perfectly fine. I'm just noting this here in case anyone else sees this and wonders why.